pkg/controllers/monitoring-stack/controller.go

```diff
@@ -130,20 +131,74 @@ func (r *reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		return ctrl.Result{}, nil
 	}
 
+	availableCondition := metav1.Condition{
+		Type:               "Available",
+		ObservedGeneration: ms.ObjectMeta.Generation,
+		LastTransitionTime: metav1.Time{Time: time.Now().UTC()},
+	}
+
```

Comment:
```suggestion
```
IMHO construction of the condition should move to a different functions. say 
`new[Failed|Unknown|Ready]Condition(available, msg, ms.ObjectMeta.Generation)` 
where `available` is a `const available = "Available"`